keyboards/viktus/sp_mini/keymaps/peott-fr/keymap.c

```diff
@@ -0,0 +1,114 @@
+ /* Copyright 2021 peott
+  *
+  * This program is free software: you can redistribute it and/or modify
+  * it under the terms of the GNU General Public License as published by
+  * the Free Software Foundation, either version 2 of the License, or
+  * (at your option) any later version.
+  *
+  * This program is distributed in the hope that it will be useful,
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  * GNU General Public License for more details.
+  *
+  * You should have received a copy of the GNU General Public License
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  */
+
+#include QMK_KEYBOARD_H
+
+// Layers by name
+enum custom_layers {
+    _BASE,
+    _LHAND,
+    _RHAND,
+    _NUM
+};
+
+// Layers defined by function
+#define LHAND LT(_LHAND,KC_SPC)
+#define RHAND LT(_RHAND,KC_BSPC)
+#define NUM TG(3)
```

Comment:
```suggestion
#define NUM TG(_NUM)
```
This is a little easier to read and if you change the enum this will stick with the _NUM layer.

Reply:
Thanks! This indeed fixed it :)